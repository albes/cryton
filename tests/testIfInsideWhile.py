a = 1
b = 2
c = 3

if a == 1:
    d = 99

while(a < 100):
    a = a + 1
    
    if a == c:
        d = 999999
    if a < 7:
        c = c + c


print a
print c
print d