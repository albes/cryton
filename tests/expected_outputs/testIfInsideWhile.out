100
96
999999